### ISSUE TITLE
AddSoul not working on latest table and cheat engine

### ISSUE THREAD

**Issue by Wicedrissa** [The-Grand-Archives/Elden-Ring-CT-TGA#89]
I go to click add soul because I don't want to sit there for like 9 hours constantly using lord's runes and it won't open unless I enable the bottom one on the script and disable the top one on the script

**Comment by inuNorii** [comment]
What are you talking about with "top" and "bottom" on the script?
Are you even using the table you posted this issue for?
This script still works correctly.

**Comment by Wicedrissa** [comment]
I'm using the latest version of the script
![Cheat Engine 7 5 7_20_2024 6_31_29 PM](https://github.com/user-attachments/assets/0fcd210b-8ebe-4287-811a-2c15d67981bb)
and I'm talking about this that I have to fuck with

**Comment by inuNorii** [comment]
Well, there seems to be something wrong on your end.
When it fails to open, could you right-click on the script and check if the first entry in the context menu says anything about an error?
If yes, click on that and it will open the script and also copy the error text to your clipboard. You can paste and send that error text here.

**Comment by Wicedrissa** [comment]
k it says <<error while scanning for AOB's : AddSoul_Call Error: Not all results found>> when I right-click it

**Comment by Wicedrissa** [comment]
![Cheat Engine 7 5 7_20_2024 6_35_55 PM](https://github.com/user-attachments/assets/45007d5a-6b2e-405a-8082-3a33a986c28f)
also, here's the other half

**Comment by Wicedrissa** [comment]
also I noticed in the aobscanmodule on line 3 there are two ?? in the string

**Comment by inuNorii** [comment]
`??` in AOBs is a placeholder, necessary in case the actual byte changes.

**Comment by Wicedrissa** [comment]
ah figured

**Comment by inuNorii** [comment]
> k it says <<error while scanning for AOB's : AddSoul_Call Error: Not all results found>> when I right-click it

So for some reason it can't find this AOB for you... sometimes?
Are you playing with any mods?

**Comment by Wicedrissa** [comment]
seamless co-op that's it

**Comment by inuNorii** [comment]
Alright I'm gonna have to test that then. Wouldn't be the first script I need to adjust for seamless.

**Comment by Wicedrissa** [comment]
I'll switch my save file to the offline mod and see if there's no issue

**Comment by Wicedrissa** [comment]
yep no issues on the offline mod, it's just the seamless co-op having issues

**Comment by inuNorii** [comment]
Weird. I can't see any changes to the relevant code with Seamless active.
Is it only happening under specific circumstances?
I'm gonna ask Yui if she's changing this function in any way.
